storage/include/blockchain/block_helper.h

```diff
@@ -0,0 +1,44 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "db_types.h"
+#include "kv_types.hpp"
+#include "sliver.hpp"
+#include "hash_defs.h"
+
+#include <cstdint>
+#include <utility>
+
+namespace concord {
+namespace storage {
+namespace blockchain {
+inline namespace v1_direct_key_value {
+
+// A helper class used to abstract away block storage details.
+struct BlockHelper {
```

Comment:
I'd prefer if we could make this class name a bit more concrete. It's just personal taste, but I don't really like names such as `*Helper` and `*Wrapper`. I don't feel they add much value, and just lead to more typing. 

I just did a quick search and it looks like we don't actually have a type called `Block`, so we could just use that as the name. Then we can also shorten the names of these static functions to things like: `Block::create` and `Block::getData`. `getParentDigest` is a nice name still :)

Alternatively, since these are all static functions we could just not wrap them in a type but instead tie them to the namespace directly.

Reply:
I agree, using a namespace is better, doing that. I am also using the names you suggest - I think they are simple and descriptive enough.